### PR TITLE
Add v2.3.0 release notes

### PR DESCRIPTION
**Pull request by noCharger**
Signed-off-by: Louis Chu <clingzhi@amazon.com>

### Description
Add v2.3.0 release note
 
### Issues Resolved
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2299
 
### Check List
- [x] New functionality has been documented.
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by ashwin-pc — DISMISSED** [review]

**Review by ananzh — DISMISSED** [review]

File: release-notes/opensearch-dashboards.release-notes-2.3.0.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- [D&D] Persist index field on agg type change if possible ([#2227](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2227))
+- [D&D] Add count field to field picker ([#2231](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2231))
+- [D&D] Add Bar line and Area charts to Wizard ([#2266](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2266))
+
+### 🐛 Bug Fixes
+
+- Fix maps wms zoom limitation ([#1915](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1915))
+- [Visualizations] Add visConfig.title and uiState to build pipeline function (([#2192](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2192))
+- Custom healthcheck with filters ([#2232](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2232))
+- [BUG] fix healthcheck logic to expect object and return ids ([#2277](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2277))
```
Comment (vagimeli) [inline release-notes/opensearch-dashboards.release-notes-2.3.0.md:17]: Capitalize "Fix"

File: release-notes/opensearch-dashboards.release-notes-2.3.0.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+### 🚞 Refactor
+
+- [DeAngular][visualization][vislib] remove angular from vislib ([#2138](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2138))
```
Comment (vagimeli) [inline release-notes/opensearch-dashboards.release-notes-2.3.0.md:21]: Capitalize "Remove"

File: release-notes/opensearch-dashboards.release-notes-2.3.0.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- Change timeline icon ([#2162](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2162))
+
+### 🛠 Maintenance
+
+- increment from 2.2 to 2.3. ([#2096](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2096))
```
Comment (vagimeli) [inline release-notes/opensearch-dashboards.release-notes-2.3.0.md:26]: Capitalize "Increment"

File: release-notes/opensearch-dashboards.release-notes-2.3.0.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+### 🔩 Tests
+
+- [CI][tests] add BWC tests for 2.3.0 ([#2281](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2281))
```
Comment (vagimeli) [inline release-notes/opensearch-dashboards.release-notes-2.3.0.md:30]: "Capitalize "Add" and define BWC. For example, "Add backwards compatibility (BWC) tests..."

File: release-notes/opensearch-dashboards.release-notes-2.3.0.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+- fix 2.x bwc by restoring 1.3.2 ([#2284](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2284))
```
Comment (vagimeli) [inline release-notes/opensearch-dashboards.release-notes-2.3.0.md:31]: Capitalize "Fix" and "BWC"

**Review by abbyhu2000 — APPROVED** [review]

**Comment by ashwin-pc** [comment]
@noCharger can you add a signoff to the second commit. It fails DCO

**Comment by noCharger** [comment]
> @noCharger can you add a signoff to the second commit. It fails DCO

updated

**Review by vagimeli — APPROVED** [review]
LGTM

**Review by abbyhu2000 — APPROVED** [review]

**Review by ananzh — APPROVED** [review]

**Review by ashwin-pc — APPROVED** [review]

**Pull request merged**